components/Tabs/Tabs.js

```diff
@@ -47,10 +44,10 @@ const Tabs = (props) => {
           <TabPanel value={value} index={index} key={index}>
             {props.data
               .filter((item) => new Date(item.date).getFullYear() === year)
-              .map((item) => {
+              .map((item, index) => {
                 return (
                   <EventItem
-                    key={item.id}
+                    key={index}
```

Comment:
actually using `id` is recommended than using `index.

Reply:
but I get an error message saying `Each child in a list should have a unique "key" prop.` when I keep id.🧐

Reply:
hm, what's in `item.id`? sounds like it's all `undefined`. Is there some value in `item` that is good for an id?